src/NetVar.cc

```diff
@@ -108,6 +108,8 @@ zeek::RecordType* dns_answer;
 zeek::RecordType* dns_soa;
 zeek::RecordType* dns_edns_additional;
 zeek::RecordType* dns_edns_ecs;
+zeek::RecordType* dns_edns_tcp_keepalive;
```

Comment:
These variables aren't necessary since we'd just be deleting them in v4.1 anyways. The others are all marked deprecated because they existed before we made other changes.